config/suppress.xml

```diff
@@ -4,4 +4,5 @@
         "http://www.puppycrawl.com/dtds/suppressions_1_0.dtd">
 <suppressions>
     <suppress files=".*generated.*" checks="."/>
+    <suppress files="CppGenerator.java" checks="MethodLength"/>
```

Comment:
I don't want to split up the constructor codegen function for no good reason. Should I do something like this:
```
void generateConstructor() {
  generatePart1();
  generatePart2();
}
```